src/aks-preview/HISTORY.rst

```diff
@@ -13,6 +13,10 @@ Pending
 +++++++
 * Vendor new SDK and bump API version to 2024-07-02-preview.
```

Comment:
Please include this line in your newly added version.

Reply:
updated.